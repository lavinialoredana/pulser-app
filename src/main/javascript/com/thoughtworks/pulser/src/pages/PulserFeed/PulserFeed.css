.Pulser-feed {
    text-align: center;
    display: flex;
    flex-direction: column;
}

.Pulser-feed-header {
    background-color: #282c34;
    min-height: 10vh;
    width: 100vw;
    font-size: calc(10px + 2vmin);
    color: white;
}