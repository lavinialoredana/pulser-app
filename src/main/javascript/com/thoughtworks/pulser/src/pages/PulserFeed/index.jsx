import "./PulserFeed.css";

const PulserFeed = () => {
  return (
    <div className="Pulser-feed">
      <header className="Pulser-feed-header" data-testid = "pulser-feed"> 
      <h1> Pulser Feed</h1>
      </header>
    </div>
  );
};

export default PulserFeed;
