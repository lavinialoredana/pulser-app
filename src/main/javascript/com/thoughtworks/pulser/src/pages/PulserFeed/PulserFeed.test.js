import { cleanup, render, screen } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import PulserFeed from "./index.jsx";

describe("PulserFeed Page structure", () => {
  afterEach(() => {
    cleanup();
  });

  it("should match snapshot", () => {
    const { asFragment } = render(<PulserFeed />, { wrapper: BrowserRouter });

    expect(asFragment()).toMatchSnapshot();
  });

  it("should render the title", () => {
    render(<PulserFeed />, { wrapper: BrowserRouter });

    const titleElement = screen.getByTestId("pulser-feed");

    expect(titleElement).toBeInTheDocument();
  });
});
