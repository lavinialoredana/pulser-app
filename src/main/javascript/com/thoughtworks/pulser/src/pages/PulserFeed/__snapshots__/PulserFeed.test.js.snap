// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PulserFeed Page structure should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Pulser-feed"
  >
    <header
      class="Pulser-feed-header"
      data-testid="pulser-feed"
    >
      <h1>
         Pulser Feed
      </h1>
    </header>
  </div>
</DocumentFragment>
`;
